plugins/kotlin/fir/src/org/jetbrains/kotlin/idea/completion/contributors/FirCallableCompletionContributor.kt

```diff
@@ -44,8 +50,9 @@ internal open class FirCallableCompletionContributor(
         is KtExtensionApplicabilityResult.NonApplicable -> null
     }
 
-    protected fun KtAnalysisSession.getOptions(symbol: KtCallableSymbol): CallableInsertionOptions =
-        CallableInsertionOptions(detectImportStrategy(symbol), getInsertionStrategy(symbol))
+    protected fun KtAnalysisSession.getOptions(symbol: KtCallableSymbol): CallableInsertionOptions? {
+        return CallableInsertionOptions(detectImportStrategy(symbol), getInsertionStrategy(symbol) ?: return null)
```

Comment:
Please, do not use inline `return/break/continue` statements. Introduce a variable for that

Reply:
OK. After the refactoring I no longer need to make `getOptions` return nullable values.